CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 ### HEAD
 
+#### Features
+
+* Added metadata to the gem (such as changelog URI, source code URI & bug
+  tracker URI), so it shows on https://rubygems.org/gems/pry
+  ([#1869](https://github.com/pry/pry/pull/1869))
+
+#### API changes
+
+* `Pry::Prompt` is a class now and it can be instantiated to create new prompts
+  on the fly that are not registered with `Pry::Prompt#add`. Learn more about
+  its API in the docs ([#1877](https://github.com/pry/pry/pull/1877))
+
+#### Deprecations
+
+* Deprecated `Pry.config.exception_whitelist` in favor of
+  `Pry.config.unrescued_exceptions`
+  ([#1874](https://github.com/pry/pry/pull/1874))
+* Deprecated `Pry.config.prompt = Pry::Prompt[:simple][:value]` in favor of
+  `Pry.config.prompt = Pry::Prompt[:simple]` when setting Pry prompt via
+  `pryrc` ([#1877](https://github.com/pry/pry/pull/1877))
+* Deprecated setting prompt via an array of two procs:
+  ([#1877](https://github.com/pry/pry/pull/1877))
+
+  ```ruby
+  # Deprecated, emits a warning.
+  Pry.config.prompt = [proc {}, proc {}]
+  ```
+
+  This will be removed in the next release.
+
+#### Breaking changes
+
+* Deleted deprecated `Pry::Platform`
```

Comment:
we only deprecated these constants/modules on nov 12 right? maybe we need to give more time to allow people to stop using them and update their code, if we're releasing v0.13 very soon

Reply:
Yeah, it's a recent change. That said, I don't think 0.13 will be any time soon, so we're good.